dataservices/dataservices.go

```diff
@@ -75,6 +89,11 @@ func (client *DataServiceClient) Run(URL string) error {
 	}
 	client.api.SetApp(router)
 
+	if serviceConfig.Protocol == "https" {
+
+		return http.ListenAndServeTLS(URL, serviceConfig.CertFile, serviceConfig.KeyFile, client.api.MakeHandler())
+		//return server.ListenAndServeTLS(URL, serviceConfig.CertFile, serviceConfig.KeyFile, client.api.MakeHandler())
```

Comment:
no uncommented code without a specific TODO indicating why it is commented out yet still in the code.